controllers/app.controllers.js

```diff
@@ -82,3 +83,14 @@ exports.patchReviewById = (req, res, next) => {
       next(err);
     });
 };
+
+exports.deleteCommentById = (req, res, next) => {
+  const commentId = req.params.comment_id;
+  deleteComment(commentId)
+    .then((comment) => {
+      res.status(204).send({ comment });
```

Comment:
204 wont actually send any content, so the send isnt doing anything. 
Line 90, you also dont need the comment, as theres nothing to send back. 
it would be better as `res.sendStatus(204)` - then you dont need the send bit